accord-core/src/main/java/accord/txn/Txn.java

```diff
@@ -4,12 +4,15 @@
 
 import accord.api.*;
 import accord.local.*;
+import accord.primitives.Keys;
+import accord.primitives.Timestamp;
 
 public class Txn
 {
-    enum Kind { READ, WRITE, RECONFIGURE }
```

Comment:
removing `RECONFIGURE` for now or is this now handled differently?  Want to make sure I don't miss.

Reply:
It was introduced in a very early iteration and never ended up making sense, I had just failed to notice it hadn't been removed.